lib/lint.ml

```diff
@@ -35,10 +61,18 @@ module Make (Docker : S.DOCKER_CONTEXT) = struct
     in
     Docker.build ~label:"fmt" ~dockerfile source
 
+  (** An image with ODoc installed *)
+  let odoc_base_image ~base ~opam_files ~source =
+    let dockerfile =
+      let+ base = base and+ opam_files = opam_files in
+      `Contents (odoc_dockerfile ~base ~opam_files)
+    in
+    Docker.build ~label:"ODoc" ~dockerfile source
```

Comment:
Nitpick: On https://github.com/ocaml/odoc, it's very rarely capitalized and when it is, it's `Odoc`.